collect_app/src/main/AndroidManifest.xml

```diff
@@ -133,7 +133,10 @@ the specific language governing permissions and limitations under the License.
         <activity android:name=".preferences.screens.ProjectPreferencesActivity" />
         <activity android:name=".activities.FormHierarchyActivity" />
         <activity android:name=".activities.ViewOnlyFormHierarchyActivity" />
-        <activity android:name=".activities.GeoPointActivity" />
+        <activity
+            android:name="org.odk.collect.geo.GeoPointActivity"
+            android:theme="@style/Theme.Collect"
+            tools:replace="android:theme"/>
```

Comment:
This is a pretty nifty bit of support for Android libraries that lets us override manifest values and specify priority - here `collect_app` is saying it overrides whatever the original `GeoPointActivity` theme is.